src/app/api/organizations/route.ts

```diff
@@ -0,0 +1,24 @@
+import { createOrganization } from '@/server/actions/Organizations';
+import { NextRequest, NextResponse } from 'next/server';
+
+// @route Post /api/organizations/ - Creates an organization
+
+export async function POST(request: NextRequest) {
+  try {
+    const req = await request.json();
```

Comment:
we'll want to validate the request body to make sure it is in the right format before sending it to the server func. [Here's](https://github.com/hack4impact-utk/compassion-ministries/blob/main/src/app/api/organizations/%5BorganizationId%5D/route.ts) an example, except this will be validating against `zCreateOrganizationRequest`. One thing to add to that (and I just learned this recently!) is that the return value of the `safeParse` function has a `success` prop and a `data` prop. The `data` prop is what we should be passing into the server func, as it strips away any extra fields (extra fields do not throw validation errors, they are just not validated)

Reply:
sounds good